Analyzer/src/MakeMiniTree.cc

```diff
@@ -10,110 +10,195 @@ MakeMiniTree::MakeMiniTree()
     InitHistos();
 }
 
-
 void MakeMiniTree::InitHistos()
 {
-    my_histos.emplace( "EventCounter", std::make_shared<TH1D>( "EventCounter", "EventCounter", 2, -1.1, 1.1 ) ); 
-}//END of init histos
+    eventCounter = std::make_shared<TH1D>("EventCounter", "EventCounter", 2, -1.1, 1.1);
+}
 
 void MakeMiniTree::Loop(NTupleReader& tr, double, int maxevents, bool)
 {
     while( tr.getNextEvent() )
     {
-        const auto& eventCounter        = tr.getVar<int>("eventCounter");
-        my_histos["EventCounter"]->Fill( eventCounter );
-
-        const auto& runtype             = tr.getVar<std::string>("runtype");
-        const auto& passBaseline1l      = tr.getVar<bool>("passBaseline1l_Good");
-       
         //------------------------------------
         //-- Print Event Number
         //------------------------------------
-        if( maxevents != -1 && tr.getEvtNum() >= maxevents ) break;
-        if( tr.getEvtNum() % 10000 == 0 ) printf( " Event %i\n", tr.getEvtNum() );
+        if( maxevents != -1 && tr.getEvtNum() > maxevents )
+            break;
+        if( tr.getEvtNum() % 1000 == 0 )
+            printf( " Event %i\n", tr.getEvtNum() );
+
+        eventCounter->Fill(1.0);
```

Comment:
Why are you filling 1.0 here? There was a reason why I made a variable called `eventCounter`

Reply:
It was to take into account the negative weighted events properly. This change impacts the hadder.py script too because that is where this `EventCounter` histogram is used.

Reply:
lol. np.

Reply:
Resolved in #65208e0